src/bot.ts

```diff
@@ -92,7 +97,7 @@ export default (app: Probot) => {
   app.on('issue_comment.created', async (context: Context<'issue_comment.created'>) => {
     const repo = context.payload.repository.name;
     const owner = context.payload.repository.owner.login;
-    const organization = context.payload.repository.owner.login;
+    const organization = context.payload.organization?.login ?? '';
```

Comment:
i dont believe this one needs to change actually. `context.payload.repository.owner.login` should always exist - https://api.github.com/repos/gitpoap/gitpoap-bot

another way i'm thinking about it.. according to the type returned by octokit for context, `context.payload.repository.owner.login` should always be a string. 

With this in mind, i feel that setting line 27 to `context.payload.repository.owner.login` would be a better choice than to do `const organization = context.payload.organization?.login ?? '';`

**_also, one thing to consider is that not all repos are owned by organizations, but are instead owned by users -> i believe this is the root of the issue._**

Reply:
@peebeejay do you think its enough to add validation for organization?

Reply:
what do you specifically mean by add validation?

Reply:
![image](https://user-images.githubusercontent.com/110413633/199522980-9ba2ace1-f6ea-4f0d-907d-ebe240154343.png)

Reply:
if you mean checking if an organization / owner name exists, then yes, i think that's fine to keep (although the types suggest it'll never be empty). 

if we're gonna have the if statements, it would be good to dispatch an error to sentry so that we see this stuff though - that would involve using `context.log.error` instead of `context.log.info`.